strawberry/custom_scalar.py

```diff
@@ -32,7 +32,7 @@
     _T = TypeVar("_T", bound=type)
 
 
-def identity(x):
+def identity(x) -> Any:
```

Comment:
can you type this as:

```

def identity(x: T) -> T:
    return x
```

https://mypy-play.net/?mypy=latest&python=3.11&gist=fcf488a41be8a6f87e3aef4a0d188818